python/ray/tune/examples/cifar10_pytorch.py

```diff
@@ -187,21 +186,18 @@ def main(num_samples=10, max_num_epochs=10, gpus_per_trial=2):
         "batch_size": tune.choice([2, 4, 8, 16])
     }
     scheduler = ASHAScheduler(
-        metric="loss",
-        mode="min",
         max_t=max_num_epochs,
         grace_period=1,
         reduction_factor=2)
-    reporter = CLIReporter(
-        # parameter_columns=["l1", "l2", "lr", "batch_size"],
-        metric_columns=["loss", "accuracy", "training_iteration"])
     result = tune.run(
-        partial(train_cifar, data_dir=data_dir),
+        tune.with_parameters(train_cifar, data_dir=data_dir),
         resources_per_trial={"cpu": 2, "gpu": gpus_per_trial},
         config=config,
+        metric="loss",
+        mode="min",
         num_samples=num_samples,
-        scheduler=scheduler,
-        progress_reporter=reporter)
+        scheduler=scheduler
+    )
```

Comment:
```suggestion
        scheduler=scheduler)
```
inb4 linter

Reply:
unfortunately the linter is keeping it as is

Reply:
My bad!